.gitignore

```diff
@@ -1,21 +1,42 @@
-.nyc_output
-coverage
-dist
-lerna-debug.log
+# Package manager
 node_modules
-npm-debug.log
-yarn-error.log
+
+# Build
+dist
 _webpackOutput
 
 # Generated metadata (only the root one is generated!)
-
 /metadata.toml
 
+# Cache
+.cache
+.eslintcache
+
+# Testing
+.nyc_output
+coverage
 
-# Phenomic things
+# Env
+.env.local
+.env.*.local
 
-*.bundle.js
+# Logging
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+lerna-debug.log*
+.pnpm-debug.log*
 
-# Cache
+# Editor
+.idea
+.vscode
+.history
+*.suo
+*.ntvs*
+*.njsproj
+*.sln
+*.sw?
 
-.eslintcache
```

Comment:
Better to keep it, because eslint generate cache

Reply:
It's just been moved to another location in the file, under the `# Cache` section :-)